api/proto/holomush/web/v1/web.proto

```
@@ -0,0 +1,63 @@
+option go_package = "github.com/holomush/holomush/pkg/proto/holomush/web/v1;webv1";
+
+service WebService {
+  // Authenticate as guest or registered user.
+  rpc Login(LoginRequest) returns (LoginResponse);
+
+  // Send a game command (say, pose, quit, etc.)
+  rpc SendCommand(SendCommandRequest) returns (SendCommandResponse);
+
+  // Server-streaming event feed. Client receives game events
+  // (say, pose, arrive, leave) as they occur.
+  rpc StreamEvents(StreamEventsRequest) returns (stream GameEvent);
```

Buf (1.66.1)

[error] RPC response type "GameEvent" should be named "StreamEventsResponse" or "WebServiceStreamEventsResponse".

(RPC_RESPONSE_STANDARD_NAME)